assets/lokomotive-kubernetes/packet/flatcar-linux/kubernetes/workers/workers.tf

```diff
@@ -20,22 +20,16 @@ resource "packet_device" "nodes" {
   tags = var.tags
 }
 
-data "ct_config" "install-ignitions" {
-  content = data.template_file.install.rendered
-}
-
 # These configs are used for the fist boot, to run flatcar-install
-data "template_file" "install" {
-  template = file("${path.module}/cl/install.yaml.tmpl")
-
-  vars = {
+data "ct_config" "install-ignitions" {
```

Comment:
Don't see the need to split both commits in workers and controllers, if they are basically doing the same thing. IMHO, it is a little harder to review just checking if the same was done in the other commit.

Reply:
True, will squash.

Reply:
Thanks!

Reply:
Done